cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -243,11 +253,19 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
   function getDom() {
     return {
       altTrigger: _altTriggerDom,
+      container:  _dom,
       content:    _contentDom,
       trigger:    _triggerDom
     };
   }
 
+  /**
+   * @returns {boolean} True if menu is expanded, false otherwise.
+   */
+  function isAnimating() {
```

Comment:
Consider using a utility `Object.defineProperty` function for places where you want public getters to access private data. The benefit is a property check instead of a function call and write/read control.

https://github.com/cfpb/cfgov-refresh/pull/1566/files#diff-7a844d22219d7d3db1fa7c1e70d7ba45R35

Shouldn't be addressed in this PR.